.gitignore

```diff
@@ -126,4 +126,10 @@ iOSInjectionProject/
 /*.gcno
 **/xcshareddata/WorkspaceSettings.xcsettings
 
+# R.swift
+#
+# R.generated.swift file will be automatically genrated by R.swift
+Yumemi-ios-training/Yumemi-ios-training/GeneratedByTools/Rswift/R.generated.swift
```

Comment:
こう書いて良さそうです。

```suggestion
*.generated.swift
```


> Tip: Add the *.generated.swift pattern to your .gitignore file to prevent unnecessary conflicts.
> 
> https://github.com/mac-cain13/R.swift